hazelcast/src/test/java/com/hazelcast/map/impl/query/QueryNullIndexingTest.java

```diff
@@ -83,9 +83,9 @@ public void testIndexedNullValueOnUnorderedIndexStoreWithNotEqualPredicate() {
         final List<Long> dates =
                 queryIndexedDateFieldAsNullValue(false,
                                                  Predicates.notEqual("date", 2000000L));
-        assertEquals(4, dates.size());
+        assertEquals(9, dates.size());
         assertTrue(dates.containsAll(
-                Arrays.asList(4000000L, 6000000L, 8000000L, 10000000L)));
+                Arrays.asList(4000000L, 6000000L, 8000000L, 10000000L, null)));
```

Comment:
this is a side-effect of this change. "notEquals(attribute, not-NULL)" not returns values where the `attribute` is set to null.